main/src/ca/uwaterloo/flix/language/phase/Safety.scala

```diff
@@ -1073,4 +1083,13 @@ object Safety {
   private def isAbstractMethod(m: java.lang.reflect.Method): Boolean =
     java.lang.reflect.Modifier.isAbstract(m.getModifiers)
 
+  /**
+    * Returns the control-effects inside `tpe`.
+    */
+  private def getControlEffs(tpe: Type): Set[Symbol.EffectSym] =
```

Comment:
Should this be in Type? We already have several similar functions there.

Reply:
I think this function is stupid and should be reworked. So I would keep it here for now.